src/sentry/event_manager.py

```diff
@@ -513,16 +523,14 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         # Check to make sure we're not about to do a bunch of work that's
```

Comment:
This is not accurate anymore. Our concern is now that the order by clause in Clickhouse include the timestamp of the event (rounded to the beginning of the day) so potentially, two events with the same project_id and event_id and timestamp far apart enough would not be merged by clickhouse thus creating an inconsistency in the data model.